static/src/js/utils/getUmmSchema.js

```diff
@@ -17,6 +18,8 @@ const getUmmSchema = (conceptType) => {
       return ummTSchema
     case 'Variable':
       return ummVarSchema
+    case 'Keyword':
```

Comment:
This isn't a umm schema, so I don't think this is the way we should reference the schema through this method.  Look at some other other other one-of umm schemas, my guess is they are just referenced direclty in the component.